src/pages/academy/subscription/index.tsx

```diff
@@ -54,7 +54,7 @@ const Subscription = () => {
                 </Flex>
                 <Flex>
                     <LinkButton secondary to="/academy/">
-                        {localize('Take me to Academy')}
+                        {localize(academy_link_text)}
```

Comment:
Please use `Localize` component instead of `localize` function

Reply:
> Please use `Localize` component instead of `localize` function

@mohammad-hashemi-deriv you missed this one, could u pls change this as well :pray: